### PR TITLE
fix: Moved `StatusTabButton` to StatusQ

### PR DESCRIPTION
**Pull request by igor-sirotin**
Corresponding StatusQ PR:
- https://github.com/status-im/StatusQ/pull/684

### What does the PR do

`StatusTabButton` is moved to StatusQ library.
Updated all places with `StatusTabButton` to use the new component together with `StatusTabBar` component.

### Affected areas

- chat
- wallet
- settings

### Screenshot of functionality

image

image

### Cool Spaceship Picture

image

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]
LGTM!
Only comment: This is not actually a bugfix.

File: ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml [hunk 1]
```diff
@@ -181,29 +181,22 @@ Popup {
...
-                border.width: 0
-            }
 
             StatusTabButton {
                 id: assetBtn
                 //% "Assets"
-                btnText: qsTrId("wallet-assets")
-                anchors.top: parent.top
+                width: implicitWidth
+                text: qsTrId("wallet-assets")
```
Comment (stefandunca) [inline ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml:193]: Note that we work to move away from using translation IDs. Let's refactor everytime we have a chance by adding qsTr("<User facing text here>")
Also, the `//% "Assets"` comment was related to the next line. Therefore, move this string in the `qsTr` instead.
```suggestion
                text: qsTr("Assets")
```

File: ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml [hunk 2]
```diff
@@ -181,29 +181,22 @@ Popup {
...
             }
             StatusTabButton {
                 id: historyBtn
-                anchors.top: parent.top
-                anchors.left: assetBtn.right
-                anchors.leftMargin: 32
                 //% "History"
-                btnText: qsTrId("history")
+                width: implicitWidth
```
Comment (stefandunca) [inline ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml:198]: Same note as above

**Review by stefandunca — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/ContactsView.qml [hunk 3]
```diff
@@ -51,42 +51,39 @@ SettingsContentBase {
...
-            height: contactsBtn.height
-            background: Rectangle {
-                color: Style.current.transparent
-            }
+            
             StatusTabButton {
                 id: contactsBtn
-                addToWidth: Style.current.bigPadding
-                btnText: qsTr("Contacts")
+                width: implicitWidth
```
Comment (borismelnik) [inline ui/app/AppLayouts/Profile/views/ContactsView.qml:63]: What a reason for it? `implicitWidth` should be original size and if `width` not set it should use `implicitWidth`

**Pull request merged**